src/pydata_sphinx_theme/assets/styles/extensions/_notebooks.scss

```diff
@@ -52,6 +52,28 @@ div.nblast.container {
   margin-bottom: 1rem;
 }
 
+// Notebook cell input line number
+.nbinput.container .prompt pre {
+  html[data-theme="light"] & {
+    color: #005b82;
+  }
+
+  html[data-theme="dark"] & {
+    color: #00e0e0;
+  }
+}
+
+// Notebook cell output line number
+.nboutput.container .prompt pre {
+  html[data-theme="light"] & {
+    color: #a12236;
+  }
+
+  html[data-theme="dark"] & {
+    color: #ffa07a;
+  }
+}
```

Comment:
TODO: check MyST-NB notebook prompts for color contrast

Reply:
Actually MyST-NB does not render notebook cell prompts:

<img width="979" alt="screenshot" src="https://github.com/pydata/pydata-sphinx-theme/assets/317883/554d6ae3-0862-4434-8e59-9531f26900da">